index.html

```diff
@@ -1,51 +1,9 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <meta name="description" content="">
-    <meta name="author" content="">
-    <link rel="icon" href="img/favicon.png">
-
-    <title>ReFrame</title>
-
-    <!-- Bootstrap core CSS -->
-    <link href="css/bootstrap.min.css" rel="stylesheet">
-
-    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
-    <link href="css/viewport-hack-surface-windows8.css" rel="stylesheet">
-
-    <!-- Custom styles for this template -->
-    <link href="css/fixed-nav-bar-footer.css" rel="stylesheet">
-    <link href="css/remark-corrections.css" rel="stylesheet">
-
-    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
-    <!--[if lt IE 9]>
-      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
-      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
-    <![endif]-->
+      <meta http-equiv='refresh' content='0; URL=https://reframe-hpc.readthedocs.io/en/stable/'>
   </head>
-  <body data-spy="scroll">
-    <div id="cscs-site-content">
-    </div>
-    <!-- JavaScript core
-    ================================================== -->
-    <!-- Placed at the end of the document so the pages load faster -->
-    <script src="js/marked.min.js"></script>
-    <!-- <script src="js/remark.js"></script> -->
-    <script src="js/prism.js"></script>
-    <script src="js/jquery-3.1.1.js"></script>
-    <script src="js/bootstrap.min.js"></script>
-    <script src="js/jquery-toc.min.js"></script>
-    <script src="js/reframe.min.js"></script>
-    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
-    <script src="js/viewport-hack-surface-windows8.js"></script>
-
-    <script>
-      reframe.setup_site_content("html/base.html", "SIDEBAR.md", "JUMBOTRON.md");
-      reframe.setup_markdown_page_content("README.md");
-    </script>
+  <body>
+      <a href='https://reframe-hpc.readthedocs.io/en/stable/'>Click here in case auto redirect is not working!</a>
```

Comment:
Could you try without the body. My browser actually renders this page and it's not very nice.